stanford-sw/test/data/managedPurlTests.xml

```diff
@@ -60,7 +60,7 @@
       <subfield code="x">DDR-PURL</subfield>
       <subfield code="x">item</subfield>
       <subfield code="x">image</subfield>
-      <subfield code="x">file:file1.jpg, file2.jpg</subfield>
```

Comment:
i'm kinda muzzy on all this, but it *seems* like a good idea to have at least one example with multiple files, whether here or inline data somewhere.